api/secret.go

```diff
@@ -189,6 +172,23 @@ DONE:
 	return policies, nil
 }
 
+// TokenAccessors returns the standardized list of accessors for the given secret
+func (s *Secret) TokenAccessors() ([]string, error) {
```

Comment:
You name this `TokenAccessors`, but it could just as well be called `Keys` or something, right?  It's the same code that would be used to extract the result of any `list` operation from a `Secret`.

Reply:
Yes, you are right on that. I removed the if statement that would return the secret's accessor value because that was wrong. Instead the function call basically just invokes the `extractListOfStrings` with the correct parameters.